building/userland_integration.rst

```diff
@@ -288,49 +290,53 @@ OP-TEE contained key:
 
 .. code-block:: none
 
-    # OPENSSL_CONF=optee_hsm.conf openssl req -new -engine pkcs11 -keyform engine -key label_mykey -subj "/CN=My CSR" -x509 -out mykey_selfsigned_cert.pem
+    # OPENSSL_CONF=optee_hsm.conf openssl req -new -engine pkcs11 -keyform engine -key 1234 -subj "/CN=My CSR" -x509 -out mykey_selfsigned_cert.pem
     engine "pkcs11" set.
 
 Again we can review this self-signed certificate:
 
 .. code-block:: none
 
-    $ openssl x509 -in mykey_selfsigned_cert.pem -text
+    $ openssl x509 -in mykey_selfsigned_cert.pem -text -noout
     Certificate:
         Data:
-            Version: 1 (0x0)
+            Version: 3 (0x2)
             Serial Number:
-                3f:8f:c8:c0:de:a8:75:ca:9d:62:79:31:c2:6c:48:f4:fd:50:22:1d
+                25:00:b4:69:fd:e8:b2:7b:21:73:d7:84:87:6b:60:ff:cc:50:ea:ca
             Signature Algorithm: ecdsa-with-SHA256
             Issuer: CN = My CSR
             Validity
-                Not Before: Mar 22 20:19:15 2023 GMT
-                Not After : Apr 21 20:19:15 2023 GMT
+                Not Before: Sep 23 10:28:02 2024 GMT
+                Not After : Oct 23 10:28:02 2024 GMT
             Subject: CN = My CSR
             Subject Public Key Info:
                 Public Key Algorithm: id-ecPublicKey
-                    Public-Key: (256 bit)
+                    Public-Key: (521 bit)
                     pub:
-                        04:e3:f8:9b:d3:2a:c8:10:1b:a6:75:81:5f:ba:f3:
-                        4c:4f:34:bb:7b:b2:d2:33:58:99:83:ba:d9:34:cf:
-                        a0:97:95:d5:68:11:74:77:78:d2:2b:94:e2:45:02:
-                        8d:3a:f6:af:f9:e6:ab:bb:db:3a:75:fe:14:33:18:
-                        2c:60:58:68:c7
-                    ASN1 OID: prime256v1
-                    NIST CURVE: P-256
+                        04:00:4d:1f:3a:60:22:55:7f:1c:4e:f7:40:36:79:
```

Comment:
The example relates to a self-signed certificate, so the public key data here should match those around lines 190 and 270 (in size, curve and value). I think it would better to re-generate all these output logs so that they look consistent to the reader in this documentation.

Reply:
Done

Reply:
Could you re-generate the logs? The public key here does not match the key shown at lines 197 and 271.